src/azure-cli/azure/cli/command_modules/hdinsight/_help.py

```diff
@@ -96,22 +117,22 @@
 
 helps['hdinsight monitor'] = """
     type: group
-    short-summary: Manage Azure Monitor logs integration on the HDInsight cluster.
+    short-summary: Manage Azure Monitor logs integration on an HDInsight cluster.
 """
 
 helps['hdinsight monitor enable'] = """
     type: command
-    short-summary: Enables the Azure Monitor logs integration on the HDInsight cluster.
+    short-summary: Enables the Azure Monitor logs integration on an HDInsight cluster.
```

Comment:
Please use active voice in for all description. `e.g. ``Enables```->```Enable```
The same as others.

Reply:
Thank you for your suggestions. I will change these verbs.

However please note we cannot change descriptions from SDK. I don't think it is wise to add command description for every command.